tasks/nix.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
+if [[ $EUID > 0 ]]
+then
+  echo "{\"Error\": \"Please run as root or sudo; Use '--user root' option to run it as root user or '--run-as root --sudo-password' options to run it with sudo in bolt tasks or plans.\"}"
```

Comment:
A few suggestions:

Omit "please".
Don't capitalize after the semicolon. 
"...use *the* '--user root...'"
"...*the* '--run-as root --sudo-password' options..."
Capitalize Bolt? Unless this is the command, then ignore. .

Reply:
Good stuff. Never thought about those details in a commit other than the technical stuff.
I have update the PR.